### PR TITLE
feat/tup-488:  Make scrollable border … not scrollable

### PR DESCRIPTION
**Pull request by R-Tomas-Gonzalez**
## Overview

This commit allows the border-top to stay put when scrolling the tbody up and down.

## Related

- [TUP-488](https://jira.tacc.utexas.edu/browse/TUP-488)

## Changes
- **Adds** 
```
thead :is(td, th) {
    border-bottom: var(--global-border-width--normal) solid var(--global-color-primary--dark);
}
```
to table.cms.css

- **Removes**
```
tbody > tr:first-child > :is(td, th),
:--s-paragraph-table {
  border-top: var(--global-border-width--normal) solid var(--global-color-primary--dark);
```
from table--basic.css

## Testing

1. http://localhost:3000/components/detail/o-fixed-header-table
2. **Tomas** could not test changes on tup-ui... _Doesn't know how to pull up the table on tup-ui_

## UI

| Top Border Stays When Scrolled |
| - |
| Screenshot 2023-11-17 at 11 06 42 AM |

## Notes

Adjusting border the way we did, in the thead, may create an issue. It allows the body to ever so slightly be seen behind the thead. Possible fix is making `top: -1px`. Any other ideas welcomed.

| Bug |
| - |
| Screenshot 2023-11-17 at 10 41 45 AM 
Answer: This was not just a me thing. |

| Posible Fix |
| - |
| Screenshot 2023-11-17 at 10 46 56 AM |

### CODE REVIEW

**Review by wesleyboar — CHANGES_REQUESTED** [review]

File: src/lib/_imports/objects/_objects.config.yml [hunk 1]
```diff
@@ -1,2 +1,3 @@
 context:
   subdir: "objects"
+  shouldLoadCMS: true
```
[the remark this answers is not in the document]
Comment (R-Tomas-Gonzalez) [reply src/lib/_imports/objects/_objects.config.yml:3]: Ahh this was needed to make border-bottom to appear below the t-body.

[the remark this answers is not in the document]
Comment (R-Tomas-Gonzalez) [reply src/lib/_imports/objects/_objects.config.yml:3]: Let me know and I can remove this.

**Comment by wesleyboar** [comment]
I mislead you yesterday. I'm gonna try to do this one, because I'm having trouble making sure nothing unexpectedly breaks.

**Comment by wesleyboar** [comment]
Closing in favor of #265.

File: src/lib/_imports/elements/table.cms.css [hunk 2]
```diff
@@ -20,3 +18,7 @@ tbody:not(thead + tbody) > tr:first-child > :is(td,th),
 :--s-paragraph-table p {
     padding-block: 10px;
 }
+
+thead :is(td, th) {
```
[the remark this answers is not in the document]
Comment (wesleyboar) [reply src/lib/_imports/elements/table.cms.css:22]: Resolved in #265.

**Pull request closed without merging**